examples/bri-3/src/bri/storage/readme.md

```diff
@@ -0,0 +1,5 @@
+Use the following console command to spin up the docker db
```

Comment:
lets use usual postgres port 5432:5432
also we can add here  in readme that .env  connection string should be set with these params

Reply:
5433 is to prevent a local db already running on that port from interfering.

Reply:
not sure i follow, this is local db, this project doesn't require 2 local dbs running, locally everybody can set port that works in .env file but for illustration we should use 5432 since that is default postgres port in all docs... for example, maybe i have 5 dbs container running for other projects so 5433 wont work either

 when we have alice and bob dbs we will then run 2 docker containers, one on 5432 second on 5433 for example, but now we have only one